lightkurve/search.py

```diff
@@ -231,6 +250,53 @@ def _default_download_dir(self):
 
         return download_dir
 
+    def _fetch_tesscut_path(self, target, download_dir, cutout_size):
+        """Downloads TESS FFI cutout and returns path to local file.
+
+        Parameters
+        ----------
+        download_dir : str
+            Path to location of `.lightkurve-cache` directory where downloaded
+            cutouts are stored
+        cutout_size : int or float
+            Side length of cutout in pixels
+
+        Returns
+        -------
+        path : str
+            Path to locally downloaded cutout file
+        """
+        from astroquery.mast import TesscutClass
+        tc = TesscutClass()
+
+        # Resolve SkyCoord of given target
+        coords = self._resolve_coords(target)
+        sector = int(self.table[0]['description'][-2])
```

Comment:
This looks like it will break from sector 10 onwards.  I recommend using
```
sector = self.table[0]['sequence_number']
```
See my comment at line 568 below where we can set the sequence_number to enable this.